{"name": "Verificar que un usuario puede iniciar sesión correctamente con credenciales válidas", "status": "broken", "statusDetails": {"message": "playwright._impl._errors.TimeoutError: Locator.click: Timeout 40000ms exceeded.\nCall log:\n  - waiting for locator(\"button[type='submit']\")\n\n", "trace": "  File \"/Library/Frameworks/Python.framework/Versions/3.13/lib/python3.13/site-packages/behave/model.py\", line 1329, in run\n    match.run(runner.context)\n    ~~~~~~~~~^^^^^^^^^^^^^^^^\n  File \"/Library/Frameworks/Python.framework/Versions/3.13/lib/python3.13/site-packages/behave/matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n    ~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"features/steps/login_steps.py\", line 29, in step_impl\n    context.login_page.click_element(context.login_page.login_button)\n    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Users/ceciliaperez/Desktop/DGII-Automatizacion/pages/base_page.py\", line 13, in click_element\n    self.page.locator(selector).click(timeout=timeout)\n    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^\n  File \"/Library/Frameworks/Python.framework/Versions/3.13/lib/python3.13/site-packages/playwright/sync_api/_generated.py\", line 15505, in click\n    self._sync(\n    ~~~~~~~~~~^\n        self._impl_obj.click(\n        ^^^^^^^^^^^^^^^^^^^^^\n    ...<9 lines>...\n        )\n        ^\n    )\n    ^\n  File \"/Library/Frameworks/Python.framework/Versions/3.13/lib/python3.13/site-packages/playwright/_impl/_sync_base.py\", line 115, in _sync\n    return task.result()\n           ~~~~~~~~~~~^^\n  File \"/Library/Frameworks/Python.framework/Versions/3.13/lib/python3.13/site-packages/playwright/_impl/_locator.py\", line 160, in click\n    return await self._frame.click(self._selector, strict=True, **params)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Library/Frameworks/Python.framework/Versions/3.13/lib/python3.13/site-packages/playwright/_impl/_frame.py\", line 521, in click\n    await self._channel.send(\"click\", self._timeout, locals_to_params(locals()))\n  File \"/Library/Frameworks/Python.framework/Versions/3.13/lib/python3.13/site-packages/playwright/_impl/_connection.py\", line 69, in send\n    return await self._connection.wrap_api_call(\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n    ...<3 lines>...\n    )\n    ^\n  File \"/Library/Frameworks/Python.framework/Versions/3.13/lib/python3.13/site-packages/playwright/_impl/_connection.py\", line 558, in wrap_api_call\n    raise rewrite_error(error, f\"{parsed_st['apiName']}: {error}\") from None\n"}, "steps": [{"name": "Given que estoy en la página de inicio de sesión", "status": "passed", "start": 1751746085450, "stop": 1751746086107}, {"name": "When introduzco un nombre de usuario y contraseña válidos", "status": "passed", "start": 1751746086107, "stop": 1751746086339}, {"name": "And hago clic en el botón de Login", "status": "broken", "statusDetails": {"message": "playwright._impl._errors.TimeoutError: Locator.click: Timeout 40000ms exceeded.\nCall log:\n  - waiting for locator(\"button[type='submit']\")\n\n", "trace": "  File \"/Library/Frameworks/Python.framework/Versions/3.13/lib/python3.13/site-packages/behave/model.py\", line 1329, in run\n    match.run(runner.context)\n    ~~~~~~~~~^^^^^^^^^^^^^^^^\n  File \"/Library/Frameworks/Python.framework/Versions/3.13/lib/python3.13/site-packages/behave/matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n    ~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"features/steps/login_steps.py\", line 29, in step_impl\n    context.login_page.click_element(context.login_page.login_button)\n    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Users/ceciliaperez/Desktop/DGII-Automatizacion/pages/base_page.py\", line 13, in click_element\n    self.page.locator(selector).click(timeout=timeout)\n    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^\n  File \"/Library/Frameworks/Python.framework/Versions/3.13/lib/python3.13/site-packages/playwright/sync_api/_generated.py\", line 15505, in click\n    self._sync(\n    ~~~~~~~~~~^\n        self._impl_obj.click(\n        ^^^^^^^^^^^^^^^^^^^^^\n    ...<9 lines>...\n        )\n        ^\n    )\n    ^\n  File \"/Library/Frameworks/Python.framework/Versions/3.13/lib/python3.13/site-packages/playwright/_impl/_sync_base.py\", line 115, in _sync\n    return task.result()\n           ~~~~~~~~~~~^^\n  File \"/Library/Frameworks/Python.framework/Versions/3.13/lib/python3.13/site-packages/playwright/_impl/_locator.py\", line 160, in click\n    return await self._frame.click(self._selector, strict=True, **params)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Library/Frameworks/Python.framework/Versions/3.13/lib/python3.13/site-packages/playwright/_impl/_frame.py\", line 521, in click\n    await self._channel.send(\"click\", self._timeout, locals_to_params(locals()))\n  File \"/Library/Frameworks/Python.framework/Versions/3.13/lib/python3.13/site-packages/playwright/_impl/_connection.py\", line 69, in send\n    return await self._connection.wrap_api_call(\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n    ...<3 lines>...\n    )\n    ^\n  File \"/Library/Frameworks/Python.framework/Versions/3.13/lib/python3.13/site-packages/playwright/_impl/_connection.py\", line 558, in wrap_api_call\n    raise rewrite_error(error, f\"{parsed_st['apiName']}: {error}\") from None\n"}, "start": 1751746086339, "stop": 1751746126354}, {"name": "Then debería ser redirigido al tablero con un mensaje de bienvenida \"You logged into a secure area!\"", "status": "skipped", "start": 1751746126439, "stop": 1751746126439}, {"name": "And cierro sesión", "status": "skipped", "start": 1751746126439, "stop": 1751746126439}], "attachments": [{"name": "Failure Screenshot", "source": "74082121-e66f-43db-adce-7ea8917495b3-attachment.png", "type": "image/png"}], "start": 1751746085111, "stop": 1751746126439, "uuid": "3e4c01fc-e434-49ab-b3ea-60bdcf738fbd", "historyId": "5e2016984f1b8795171c53a319a2679a", "fullName": "Funcionalidad de Inicio de Sesión de Usuario: Verificar que un usuario puede iniciar sesión correctamente con credenciales válidas", "labels": [{"name": "severity", "value": "normal"}, {"name": "tag", "value": "login_valid"}, {"name": "feature", "value": "Funcionalidad de Inicio de Sesión de Usuario"}, {"name": "framework", "value": "behave"}, {"name": "language", "value": "cpython3"}]}